src/redist/targets/GenerateLayout.targets

```diff
@@ -323,6 +330,7 @@
         <PackageVersion>$(MicrosoftWindowsDesktopAppRefPackageVersion)</PackageVersion>
         <TargetFramework>$(TargetFramework)</TargetFramework>
         <RelativeLayoutPath>packs/%(PackageName)/%(PackageVersion)</RelativeLayoutPath>
+        <DirectoriesToCopy>shared/Microsoft.WindowsDesktop.App</DirectoriesToCopy>
```

Comment:
This should be in the next `BundledLayoutComponent`. This one is the ref pack, not the shared framework.

Reply:
Done